nemo/collections/nlp/models/nlp_model.py

```diff
@@ -410,3 +522,160 @@ def save_checkpoint(self, filepath, weights_only: bool):
                     )
                     atomic_save(checkpoint, filepath)
         return None
+
+
+class NLPDDPPlugin(DDPPlugin):
+    """ DDP plugin for Pytorch Lightning. Needed to customize DDP for model parallel models.
+    """
+
+    distributed_backend = "ddp"
+
+    def __init__(
+        self,
+        parallel_devices: Optional[List[torch.device]] = None,
+        num_nodes: int = 1,
+        cluster_environment: ClusterEnvironment = None,
+        sync_batchnorm: bool = False,
+        **kwargs: Union[Any, Dict[str, Any]],
+    ) -> None:
```

Comment:
Can you add `self.__doc__ = super().__doc__` just to copy the dicstring from the original impl?